Modules/Microsoft365DSC/DSCResources/MSFT_EXOMigrationEndpoint/MSFT_EXOMigrationEndpoint.schema.mof

```diff
@@ -0,0 +1,30 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXOMigrationEndpoint")]
+class MSFT_EXOMigrationEndpoint : OMI_BaseResource
+{
+    [Key, Description("Identity of the migration endpoint.")] String Identity;
+    [Write, Description("Specifies whether to accept untrusted certificates.")] Boolean AcceptUntrustedCertificates;
+    [Write, Description("The Application ID used for authentication.")] String AppID;
+    [Write, Description("The URL of the Key Vault that stores the application secret.")] String AppSecretKeyVaultUrl;
+    [Write, Description("The authentication method for the migration endpoint.")] String Authentication;
+    [Write, Description("The type of migration endpoint."), ValueMap{"Exchange Remote", "Outlook Anywhere", "Google Workspace", "IMAP"}, Values{"Exchange Remote", "Outlook Anywhere", "Google Workspace", "IMAP"}] String EndpointType;
```

Comment:
If only one endpoint type is implemented, we should probably trim down this list.

Reply:
resolved